pkg/operation/botanist/controlplane.go

```diff
@@ -160,8 +162,10 @@ func (b *Botanist) DeployBackupInfrastructure(ctx context.Context) error {
 
```

Comment:
Since we want to preserve force-deletion annotation; revert the changes at https://github.com/gardener/gardener/pull/1204/files#diff-ac51f64ad7839f6b9db87ac98ae8ced2R166. And add the following line here:

```suggestion
if backupInfrastructure.Annotations == nil {
		backupInfrastructure.Annotations = map[string]interface{}{}
	}
	backupInfrastructure.Annotations[gardencorev1alpha1.GardenPurpose] = shootPurpose
```

Reply:
Agreed. Done.